crypto/s2n_rsa.c

```diff
@@ -29,6 +29,15 @@
 #include "utils/s2n_safety.h"
 #include "utils/s2n_blob.h"
 
+int s2n_rsa_public_key_set_from_openssl(struct s2n_rsa_public_key *s2n_rsa, RSA *openssl_rsa)
```

Comment:
Is this used or is it supposed to be used by the application? I'm confused as to what it is supposed to be doing. If it's just to be able to create an s2n_rsa_public_key for use with other API's, then maybe we could name it something like s2n_rsa_public_key_init.